.eslintrc

```diff
@@ -33,7 +33,7 @@
       "arraysInObjects": false
     }],
     "no-unused-expressions": ["error", { "allowShortCircuit": true, "allowTernary": true }],
-    "object-curly-newline": ["error", { "minProperties": 1 }],
+    "object-curly-newline": ["error", { "minProperties": 3 }],
     "object-property-newline": ["error"],
```

Comment:
@area If you have a sec, could you give your opinion about this rule change? It reformats the code pretty extensively.

Reply:
@cgewecke and @area do you have a slack or IRC or something? `gitter` perhaps?

Reply:
I don't mind :smile:. No, we don't have any of those tools set up. I suppose something like that might be useful though.